tests/acceptance/WooCommerceCustomerListCest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+require_once __DIR__ . '/../DataFactories/Settings.php';
+require_once __DIR__ . '/../DataFactories/WooCommerceProduct.php';
+require_once __DIR__ . '/../DataFactories/WooCommerceCustomer.php';
+require_once __DIR__ . '/../DataFactories/WooCommerceOrder.php';
+
+use MailPoet\Test\DataFactories\Settings;
+use MailPoet\Test\DataFactories\WooCommerceCustomer;
+use MailPoet\Test\DataFactories\WooCommerceOrder;
+use MailPoet\Test\DataFactories\WooCommerceProduct;
+
+class WooCommerceCustomerListCest {
+
+  /** @var WooCommerceProduct */
+  private $product_factory;
+
+  function _before(\AcceptanceTester $I) {
+    $I->activateWooCommerce();
+    $this->product_factory = new WooCommerceProduct($I);
+    $customer_factory = new WooCommerceCustomer($I);
+    $order_factory = new WooCommerceOrder($I);
+    $settings_factory = new Settings();
+    $settings_factory->withWooCommerceListImportPageDisplayed(true);
+    $this->product_factory->deleteAll();
```

Comment:
Do we need to run this cleanup? It slows down the tests execution and I don't see why is it needed in this test.

Reply:
You are right. I think we need to clean up only customers so that the list is not flooded.